src/main/tv/codely/scala_intro_examples/lesson_08_collections/WordAggregator.scala

```diff
@@ -1,9 +1,15 @@
 package tv.codely.scala_intro_examples.lesson_08_collections
 
 final class WordAggregator {
-  // @ToDo: Here you have your exercise!
-  // @see tv.codely.scala_intro_examples.lesson_08_collections.WordAggregatorSpec
   def aggregateWords(text: String): Map[String, Int] = {
-    Map.empty
+    if (text.isEmpty)
+      return Map.empty
```

Comment:
Same as with the `WordCounter` case. You don't need to explicitly declare the `return` statements 🙂